gc/base/MemorySpace.cpp

```diff
@@ -583,6 +583,12 @@ MM_MemorySpace::getAllTypeFlags()
  */
 uintptr_t
 MM_MemorySpace::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
+	return releaseFreeMemoryPages(env, 0);
+}
+
+uintptr_t
+MM_MemorySpace::releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t includeMemoryType)
```

Comment:
this method should pass the request to its children, hence use also the variant of the API with the parameter.